testutil/client.go

```diff
@@ -344,7 +352,11 @@ func HttpLogin(params *LoginParams) (*HttpToken, error) {
 	if !found || newRefreshJwt == "" {
 		return nil, errors.Errorf("no refresh JWT found in the output")
 	}
-
+	fmt.Println("HttpLogin: http token ", &HttpToken{UserId: params.UserID,
+		Password:     params.Passwd,
+		AccessJwt:    newAccessJwt,
+		RefreshToken: newRefreshJwt,
+	})
```

Comment:
@matthewmcneely leaving notes for myself, this needs no action from your end.

cc @joshua-goldstein

note: need to rebase